modules/ROOT/pages/proc_defining-and-creating-pipeline-tasks.adoc

```diff
@@ -38,18 +33,14 @@ spec:
     - install
 
 ----
-+
 This `Task` starts a pod and runs a container inside that pod using the `maven:3.6.0-jdk-8-slim` image to run the specified commands. It receives an input directory called `workspace-git` which contains the source code of the application.
-+
-Only the requirement for a git repository is declared on the `Task` and not a specific git repository to be used. This allows `Tasks` to be reusable for multiple `Pipeline` s and purposes.
 
-.. Create the `Task` within your project:
-+
-----
-oc create -f maven-build-task.yaml
-----
+Only the requirement for a git repository is declared on the `Task` and not a specific git repository to be used. This allows `Tasks` to be reusable for multiple `Pipelines` and purposes.
+
+[discrete]
+== Procedure
 
-. Install the `apply-manifests` and `update-deployment` tasks from the repositories, which contain a list of reusable `Tasks` for `Pipeline`s:
+. Install the `apply-manifests` and `update-deployment` tasks from the repositories, which contain a list of reusable `Tasks` for `Pipelines`:
 +
 ----
 $ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/pipeline/update_deployment_task.yaml
```

Comment:
If I understand it correctly, this docs will be versioned but you are referencing `master` brach in different repository which will change in the future. We should create a branch or a tag in `openshift/pipelines-tutorial` for every release.

Reply:
I do not see a 0.10 branch currently for pipeline tutorials. I agree, we should link to the 0.10 branch. Adding @piyush-garg

Reply:
Can we use latest instead of the version? For example: https://raw.githubusercontent.com/openshift/pipelines-tutorial/latest/pipeline/update_deployment_task.yaml

Reply:
We do the branching before next release afaik

Reply:
After discussing internally, we have decided that we will keep the links to /master for this release. For 1.0, we need to come up with a strategy on dealing with tutorial links in the docs.

If we have to change the links to the tutorial resources for each release, it would be a maintenance overhead. If we point to the master in all links, it may cause docs pointing to a unreleased/ unpublished tutorial change. Lets discuss this and finalize the strategy  for 1.0.